commands/random/coinflip.js

```diff
@@ -0,0 +1,22 @@
+const {
+  SlashCommandBuilder,
+  EmbedBuilder,
+  AttachmentBuilder,
+} = require("discord.js");
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName("coinflip")
+    .setDescription("Flips a coin!"),
+  async execute(interaction) {
+    const randomNum = Math.round(Math.random());
+    const headsOrTails = randomNum == 0 ? "Heads!" : "Tails!";
```

Comment:
Okay this is just something I wanna say that's like FYI, if you want you could do smth like
```js
randomNum > 0.5 ? "Heads!" : "Tails!"
```